deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/JobPropertiesParser.java

```diff
@@ -66,6 +66,11 @@
     @SerializedName(FAILURE_REASON_NAME)
     private String failureReason;
 
+    private static final String STORAGE_AUTHENTICATION_TYPE = "storageAuthenticationType";
+    @Expose(serialize = true, deserialize = true)
+    @SerializedName(STORAGE_AUTHENTICATION_TYPE)
```

Comment:
Shouldn't the serialized name be "storageAuthenticationType"?

Reply:
Yes that's what it is. Did I miss something?

Reply:
No, I'm just new to Java. That is a string constant that has the value we want. I get it now. :)

Reply:
A quick check of the Java repo shows you have about half of your enums as all caps, and the other half camelCased.